pkg/plugin/connector/builtin/v1/destination.go

```diff
@@ -38,61 +40,66 @@ type destinationPluginAdapter struct {
 	// logger is used as the internal logger of destinationPluginAdapter.
 	logger log.CtxLogger
 	// ctxLogger is attached to the context of each call to the plugin.
-	ctxLogger zerolog.Logger
-
-	stream *stream[cpluginv1.DestinationRunRequest, cpluginv1.DestinationRunResponse]
+	ctxLogger     zerolog.Logger
+	schemaService cschema.Service
+	stream        *stream[cpluginv1.DestinationRunRequest, cpluginv1.DestinationRunResponse]
 }
 
 var _ connector.DestinationPlugin = (*destinationPluginAdapter)(nil)
 
-func newDestinationPluginAdapter(impl cpluginv1.DestinationPlugin, logger log.CtxLogger) *destinationPluginAdapter {
+func newDestinationPluginAdapter(impl cpluginv1.DestinationPlugin, logger log.CtxLogger, schemaService cschema.Service) *destinationPluginAdapter {
```

Comment:
Ouch, this will create lots of conflicts with my PR 🙈 Oh well, can't avoid that.

Reply:
:sweat_drops: